ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+
+      Waiting for Confluent Cloud cluster to be ready and for credentials to propagate
+      ....
+
+Produce and consume records with Confluent Cloud CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Produce records
+^^^^^^^^^^^^^^^
+
+#. Run the following command to create a new Kafka topic ``demo-topic-1``:
+
+   .. code-block:: bash
+
+      ccloud kafka topic create demo-topic-1
+
+#. Produce 10 messages to topic ``demo-topic-1`` by completing the following steps:
+
+   a. Implement the following logic:
+
+      .. code-block:: text
+
+         (for i in `seq 1 10`; do echo "${i}" ; done) | \ timeout 10s
+
+   b. Run the following command:
+
+      .. code-block:: bash
+
+         ccloud kafka topic produce demo-topic-1
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      Starting Kafka Producer. ^C or ^D to exit
+      1
+      2
+      3
+      4
+      5
+      6
+      7
+      8
+      9
+      10
+
+Consume records
+^^^^^^^^^^^^^^^
+
+#. Run the following command to consume messages from topic ``demo-topic-1``:
+
+   .. code-block:: bash
+
+      ccloud kafka topic consume demo-topic-1 -b
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      Starting Kafka Consumer. ^C or ^D to exit
+      2
+      3
+      9
+      4
+      5
+      7
+      10
+      1
+      6
+      8
+
+
+Create a service account key/secret pair
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following commmand to create a new service account:
+
+   .. code-block:: bash
+
+      ccloud service-account create demo-app-3288 --description demo-app-3288 -o json
+
+
+   .. code-block:: text
+
+      {
+         "id": 104349,
+         "name": "demo-app-3288",
+          "description": "demo-app-3288"
+      }
+
+#. Create an API key and secret for the service account ``104349`` by running
+   the following account:
+
+   .. code-block:: bash
+
+      ccloud api-key create --service-account 104349 --resource lkc-x6m01 -o json
+
+   .. code-block:: text
+
+      {
+        "key": "ESN5FSNDHOFFSUEV",
+        "secret": "nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB"
+      }
+
+#. Complete the following steps to create a local configuration file
+   ``/tmp/client.config`` with |ccloud| connection information using the newly
+   created API key and secret:
+
+   a. Open the config file:
+
+      cat /tmp/client.config
+
+   b. Update  ``/tmp/client.config``  with the following parameters:
+
+      .. code-block:: text
+
+         ssl.endpoint.identification.algorithm=https
+         sasl.mechanism=PLAIN
+         security.protocol=SASL_SSL
+         bootstrap.servers=pkc-4kgmg.us-west-2.aws.confluent.cloud:9092
+         sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="ESN5FSNDHOFFSUEV" password\="nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB";
+
+   c. Wait 90 seconds for the service account credentials to propagate
```

Comment:
These should be promoted to numbered steps, not (c), (d)

Reply:
Will update in New PR